src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -443,6 +456,14 @@ public static EmbeddedPostgres start() throws IOException
             config.put("max_connections", "300");
         }
 
+        public Builder setPGStartupWait(Duration pgStartupWait) {
+            if (pgStartupWait != null && pgStartupWait.isNegative()) {
+                pgStartupWait = null;
+            }
+            Objects.requireNonNull(pgStartupWait);
```

Comment:
right now `setPGStatupWait(Duration.ofSeconds(-1))` throws `NullPointerException`, when it should either silently replace with a zero duration, or throw `IllegalArgumentException`.

Reply:
Yep, fixed